solr/core/src/test-files/solr/solr-50-all.xml

```diff
@@ -24,6 +24,7 @@
   <str name="configSetsHandler">testConfigSetsHandler</str>
   <str name="managementPath">testManagementPath</str>
   <str name="sharedLib">testSharedLib</str>
+  <str name="allowPaths">/tmp,/home/john</str>
```

Comment:
Since this file is a NamedList, shouldn't we be using `<arr` for an array of strings?

Reply:
It could, but to make it convenient to set it as a Java Sysprop, I kept it as a simple string. De  default `solr.xml` defines it as `${solr.allowPaths:}` so that `-Dsolr.allowPaths=foo,bar,baz` will work. I'm not aware of any way to specify a native array as Java Sysprop?

Reply:
ok... I suppose both could be supported provided that the consumer checked with the aid of some new utility method that doesn't exist like `Utils.toListOfString(Object, char separator) : String` but that might be a bit much.  I suspect we have other places where we trivially use a delimited string and fail to leverage NamedList.  Your call; no strong opinion here.